### DISCUSSION TITLE
Why this Whale appears on my log ?

### DISCUSSION THREAD

**Discussion by Redlumxof** [Chia-Network/chia-blockchain#5200] [Support]
2021-05-16T13:25:34.062 full_node chia.full_node.full_node: INFO     Added unfinished_block cbe94e0ee1fe829642f2f7f965ec0d797116099d2a1794c5ba8c78387ee794c3, not farmed by us, SP: 35 time: 11.626608848571777Pool pk xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
